muikku/src/main/webapp/resources/theme-muikku/sass/flex/frontpage.scss

```diff
@@ -6,7 +6,9 @@
 
   
   .fp-announcement{
-
+    div{
```

Comment:
This affects every div element under li.fp-announcement element and therefore adds double left padding unintentionally. Is it necessary to have left padding in both div's (.fp-announcement-link and .fp-announcement-info-workspace)?